src/smartbuffer.ts

```diff
@@ -998,6 +998,37 @@ class SmartBuffer {
     return this;
   }
 
+  /**
+   * Skips reading for a specified `length` from the current read position.
+   *
+   * This method is just sugar for incrementing the read offset in a chainable way.
+   *
+   * @param length { Number } The length of data to skip.
+   *
+   * @return this
+   */
+  readSkip(length: number): SmartBuffer {
+    checkLengthValue(length);
+    this.readOffset += length;
```

Comment:
Originally I had this so that it would be more similar to `writeSkip`:

```suggestion
    this.readBuffer(length); // Ignore the returned buffer value
```

But it did not throw an error if `length` went beyond the bounds of the buffer, so I changed it.

Let me know if you would prefer it to not throw an error though.